src/pages/Staking/index.tsx

```diff
@@ -78,6 +78,9 @@ const convertBlockNumbersToWeeks = (blockNumbers: number) => {
 const ZERO_VOTE_GOVERNANCE_TOKEN_AMOUNT = newMonetaryAmount(0, VOTE_GOVERNANCE_TOKEN, true);
 const ZERO_GOVERNANCE_TOKEN_AMOUNT = newMonetaryAmount(0, GOVERNANCE_TOKEN, true);
 
+// FIXME: account for transaction fees not with a hardcoded value
+const TRANSACTION_FEE_AMOUNT = newMonetaryAmount(0.01, GOVERNANCE_TOKEN);
```

Comment:
Can you add an argument here that it's not using the base but convert it correctly.

Reply:
![Capture](https://user-images.githubusercontent.com/84005068/158158095-6cbc9b70-2ff9-47ff-98dc-7c0ee430269b.PNG)
Yes, I added the third argument (true).